appinfo/info.xml

```diff
@@ -2,7 +2,7 @@
 <info xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	  xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/info.xsd">
 	<id>circles</id>
-	<name>Circles</name>
+	<name>Teams</name>
 	<summary>Bring cloud-users closer together.</summary>
 	<description><![CDATA[Circles allows people to create their own groups of people/colleagues/friends.
```

Comment:
Name needs to be changed in the description too (not only this line), right?

Reply:
Adjusted, also in the README, we can update the description once the marketing message is clearer as well I'd say.